i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-linux-snmp.md

```diff
@@ -6,240 +6,695 @@ import Tabs from '@theme/Tabs';
+| --use-ucd          | Use UCD mib for cpu average.                  |
+| --warning-average  | Warning threshold average CPU utilization.    |
+| --critical-average | Critical threshold average CPU utilization.   |
+| --warning-core     | Warning thresholds for each CPU core          |
+| --critical-core    | Critical thresholds for each CPU core         |
+
+</TabItem>
+<TabItem value="Cpu-Detailed" label="Cpu-Detailed">
+
+| Option                 | Description                                                                                                                                          |
+|:-----------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| --memcached            | Memcached server to use (only one server).                                                                                                           |
+| --redis-server         | Redis server to use (only one server). SYntax: address\\[:port\\]                                                                                        |
+| --redis-attribute      | Set Redis Options (--redis-attribute="cnx_timeout=5").                                                                                               |
+| --redis-db             | Set Redis database index.                                                                                                                            |
+| --failback-file        | Failback on a local file if redis connection failed.                                                                                                 |
+| --memexpiration        | Time to keep data in seconds (Default: 86400).                                                                                                       |
+| --statefile-dir        | Directory for statefile (Default: '/var/lib/centreon/centplugins').                                                                                  |
+| --statefile-suffix     | Add a suffix for the statefile name (Default: '').                                                                                                   |
+| --statefile-concat-cwd | Concat current working directory with option '--statefile-dir'. Useful on Windows when plugin is compiled.                                           |
+| --statefile-format     | Format used to store cache (can be: 'dumper', 'storable', 'json').                                                                                   |
+| --statefile-key        | Key to encrypt/decrypt cache.                                                                                                                        |
+| --statefile-cipher     | Cipher to encrypt cache (Default: 'AES').                                                                                                            |
```

Comment:
These options are not related to cpu-detailed, they are available for all plugins/modes.

Reply:
Those options are not available for the swap mode for example. The ones you don't see yet are available for all modes.  
(I need to write a global tab for that). Maybe I can include another tab that says: "Thoses options are available for the following modes:...."

Reply:
Ok, my bad. It's fine this way.

Reply:
Or maybe is it possible to indicate which options are technical (the cache/statefile etc) which are generic (change-perfdata etc. which is not present in the generated doc) and which are mode-specific ?

Reply:
What do you think about something like this ?

![new_doc](https://user-images.githubusercontent.com/65223458/208744033-fcd85c10-2535-494b-9bad-0d323f1f1f75.gif)